docs/openapi.yaml

```diff
@@ -4,6 +4,98 @@ info:
   description: This API provides the interface into the data to find out how much of a dog a player might be
   version: 1.0.0
 paths:
+  /character/profile:
+    get:
+      summary: Get warmane profile data
+      description: |
+        This is effectively just a passthrough to warmane.com's JSON API. It seems to have
+        pretty strict rate limits on it, so it may not always be reliable.
+      parameters:
```

Comment:
We have discussed only running the crawler if it hasn't run on a given character/realm in the past 24 hours (or whatever time period), which I guess is a form of rate limiting.
- Would it be done on the back end? 
- Would it be sufficient to mitigate warmane.com's rate limiting?

Reply:
Oh yes I totally forgot about that. I think it should be done on the backend. I just haven't implemented it yet. Crawling for results doesn't appear to be rate-limited but there is some primitive bot detection that they have. I'm more worried about the crawler DDOS'ing them.